html-aam/roles-contextual.html

```diff
@@ -45,6 +45,16 @@
 <section data-testname="el-section" aria-label="x" data-expectedrole="region" class="ex">x</section>
 <section data-testname="el-section-no-name" class="ex-generic">x</section>
 
+<!-- el-summary -->
+<summary data-testname="el-summary-orphan" class="ex-generic">x</summary>
+<details>
+  <ol>
+    <li>x</li>
+    <li>x</li>
+  </ol>
+  <summary>x</summary>
+  <summary data-testname="el-summary-not-first-summary-child-of-details" class="ex-generic">x</summary>
```

Comment:
Any idea why this one fails in WebKit with the following?
```
FAIL message: promise_test: Unhandled rejection with value: "error: Action get_computed_role failed"
```

Reply:
I think it's due to an unused `AriaUtils.verifyRolesBySelector(".ex")` in the `<script>` block since we're only testing generic stuff. I've removed it in https://github.com/web-platform-tests/wpt/pull/43859/commits/333c3df598fc45dde63f77379384c32e7d87327e , re-running CI tests now.

Reply:
It looks like there's not an issue in Gecko/Chromium (perhaps WebKit is not exposing the `<summary>` in this context as an accessible object)?